test/ably/realtimeconnection_test.py

```diff
@@ -215,3 +216,26 @@ async def test_invalid_host(self):
         assert exception.value.status_code == 400
         assert ably.connection.state == ConnectionState.DISCONNECTED
         assert ably.connection.error_reason == exception.value
+        await ably.close()
+
+    async def test_connection_state_ttl(self):
+        ably = await RestSetup.get_ably_realtime(realtime_host="iamnotahost")
```

Comment:
this test will presumable take 2 minutes to complete? i think you should be able to make it run faster by changing `Defaults.connection_state_ttl`, just make sure you set it back to the original value at the end of the test.